src/psyclone/psyir/transformations/parallel_loop_trans.py

```diff
@@ -311,14 +331,31 @@ def apply(self, node, options=None):
 
         '''
         if not options:
-            options = {}
-        self.validate(node, options=options)
-
-        verbose = options.get("verbose", False)
-        collapse = options.get("collapse", False)
-        ignore_dep_analysis = options.get("force", False)
-        list_of_names = options.get("ignore_dependencies_for", [])
-        privatise_arrays = options.get("privatise_arrays", False)
+            self.validate_options(
+                    verbose=verbose, collapse=collapse,
+                    ignore_dependencies_for=ignore_dependencies_for,
+                    privatise_arrays=privatise_arrays,
+                    sequential=sequential, **kwargs
+            )
+            # Rename the input options that are renamed in this apply method.
```

Comment:
We should probably rename them to make them consistent. Please add a TODO.

Reply:
Added a TODO (put this issue since this is all about options).